Autocoders/Python/bin/codegen.py

```diff
@@ -581,7 +581,7 @@ def generate_topology(the_parsed_topology_xml, xml_filename, opt):
                         command_elem_save.attrib["component"] = comp_name
                         command_elem_save.attrib["mnemonic"] = parameter.get_name()+ "_PRM_SAVE"
                         command_elem_save.attrib["opcode"] = "%s"%(hex(int(parameter.get_save_opcodes()[0],base=0) + comp_id))
-                        if ("comment" in list(command_elem.attrib.keys())):
+                        if ("comment" in list(command_elem_set.attrib.keys())):
```

Comment:
Should this be referencing the command_elem_save variable instead? I made this change because it was the smallest change to avoid errors, but it looks weird to reference command_elem_set here compared to the other block on line 537

Reply:
I believe so.  I am going to merge this PR and I'll take the action to correct this.